### ISSUE TITLE
[FEA] Can Jenkins show test summary for IT tests

### ISSUE THREAD

**Issue by tgravescs** [NVIDIA/spark-rapids#5256]
**Is your feature request related to a problem? Please describe.**
It would be nice if jenkins ui could show a summary of the test pass/fail like junit where it has number passed and failed, and could easily see which tests failed without having to look through log output.

One thing I remember seeing is junitxml=TEST-integration_test_result.xml.
Could se use that and integrate with  jenkins to display?

**Comment by HaoYang670** [comment]
Sorry, my mistake.

**Comment by tgravescs** [comment]
I was hoping all the integration tests could show this

**Comment by tgravescs** [comment]
bumping this up, it would be nice to get this for instance integration tests have been failing for a long time for one reason and we didn't catch that in the mean time other failures were added.  It would be really nice just to have an easy to look at summary vs having to parse the entire console output

**Comment by pxLi** [comment]
added junit-like test reports for nightly standalone IT pipelines (will take effect in today's new nightly run)
Could simply click below in specific build page
![image](https://user-images.githubusercontent.com/8086184/175204936-bb4128ba-7deb-41bd-a4ad-b5c197781435.png)
to find out details of each build

**Comment by pxLi** [comment]
I will start adding reports for CSPs ENV. This may take some time since we cannot reuse the steps to fetch reports from different CSPs

**Comment by pxLi** [comment]
test report support has been added to all nightly IT pipelines including CSP ones 
except our own yarn cluster which may require some more investigations before we can correctly collect test reports w/o break current cluster configuration.

I think we are good for now. 
Let me close this one, and if we really require IT reports from our egx yarn cluster, we can file another ticket to track, thanks!
